db/migrate/20240606080623_move_confirm_dwp_result_to_person.rb

```diff
@@ -0,0 +1,6 @@
+class MoveConfirmDWPResultToPerson < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :crime_applications, :confirm_dwp_result
```

Comment:
I think this is safe to remove as we do not pass this to review and so only draft apps may have had this value set